lessons/32-js-destructure/index.js

```diff
@@ -0,0 +1,13 @@
+const favouriteFruits =prompt(" enter your top 5 favourite fruits (in order), separated by a comma?");
+let arr = favouriteFruits.split(', ');
+console.log(arr);
+
+//Assign 3 variables for their first, second, and third favorite
+var [first, second, third,,]=arr;
+
+//Assign the rest of the fruits into a variable
+var [,,,fourth,fifth]=arr;
```

Comment:
Close! We want to assign the final results of the array (`fourth`, and `fifth` here) using the `...` (rest) operator.

Try and use the spread operator to grab the last items of the array in the first line here